tools/scripts/generic.mk

```diff
@@ -24,8 +24,12 @@ ROOT_DRIVE = C:
 TIMESTAMP = 00:00:00
 copy_fun = xcopy /F /Y /B "$(subst /,\,$1)" "$(subst /,\,$2)"
 copy_folder = xcopy /F /S /Y /C /I "$(subst /,\,$1)" "$(subst /,\,$2)"
-remove_fun = del /S /Q $(subst /,\,$1)
-remove_dir = rd /S /Q $(addsuffix ",$(addprefix ",$(subst /,\,$1)))
+remove_fun_action = del /S /Q $(subst /,\,$1)
+remove_fun = $(if $(wildcard $1),$(call remove_fun_action,$(wildcard $1)),\
+			@echo rd /S /Q $(addsuffix ",$(addprefix ",$(subst /,\,$1))))
```

Comment:
Should it be `del` instead of `rd` here?

Reply:
This is because on Windows files and folders are deleted differently: del is for files and rd is for directories.